packages/sui-widget-embedder/bin/sui-widget-embedder-build.js

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+/* eslint no-console:0 */
+const program = require('commander')
+const rimraf = require('rimraf')
+const staticModule = require('static-module')
+const minifyStream = require('minify-stream')
+const {resolve} = require('path')
+const {readdirSync, statSync, createReadStream, createWriteStream} = require('fs')
+const compilerFactory = require('../compiler/production')
+
+const WIDGETS_PATH = resolve(process.cwd(), 'widgets')
+const PUBLIC_PATH = resolve(process.cwd(), 'public')
+
+const config = require(resolve(process.cwd(), 'package.json'))['sui-widget-embedder']
+const exitWithMsg = msg => { console.log(msg); process.exit(1) }
```

Comment:
Instead of duplicating this function every time, you hace a helper that will paint it in beatyful red ❤️ 
https://github.com/SUI-Components/sui/blob/master/packages/sui-helpers/cli.js#L111

Up to you.